cloud-sql/postgres/sqlalchemy/connect_connector.py

```diff
@@ -49,9 +46,12 @@ def getconn() -> pg8000.dbapi.Connection:
             user=db_user,
             password=db_pass,
             db=db_name,
+            ip_type=ip_type,
```

Comment:
decided to move `ip_type` into the most basic usage (connect call) instead of in `Connector` initialization.